central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/editors/JBossCentralEditor.java

```diff
@@ -197,13 +199,15 @@ protected void addPages() {
 			}
 			setPageImage(index, gettingStartedImage);
 
-			softwarePage = new SoftwarePage(this);
-			index = addPage(softwarePage);
-			if (softwareImage == null) {
-				softwareImage = JBossCentralActivator.getImageDescriptor(
-						"/icons/software.png").createImage();
+			if (!new EclipseUserAgent().getApplicationName().contains("rh-eclipse")) { // Most likely RPM
```

Comment:
@nickboldt @alexeykazakov would it be better to check for `contains("|")` here? or `contains("|DevStudio")` maybe?